src/Components/Calculator.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Calculator = () => (
+
+  <>
+    <div className="container">
+      <input className="input-calc" defaultValue={0} />
+      <div className="buttons">
+
+        <CalcButton value="AC" />
+        <CalcButton value="+/-" />
+        <CalcButton value="%" />
+        <CalcButton className="operation" value="/" />
+
+        <CalcButton className="number" value={7} />
+        <CalcButton className="number" value={8} />
+        <CalcButton className="number" value={9} />
+        <CalcButton className="operation" value="x" />
+
+        <CalcButton className="number" value={4} />
+        <CalcButton className="number" value={5} />
+        <CalcButton className="number" value={6} />
+        <CalcButton className="operation" value="-" />
+
+        <CalcButton className="number" value={1} />
+        <CalcButton className="number" value={2} />
+        <CalcButton className="number" value={3} />
+        <CalcButton className="operation" value="+" />
+
+        <CalcButton className="number cero" value="0" />
+        <CalcButton value="." />
+        <CalcButton className="operation equal" value="=" />
+      </div>
+
+    </div>
+
+  </>
```

Comment:
As suggested in the project requirement, you were advised to use functional components in your project, especially the `Calculator` component. Kindly implement these changes to meet the project requirement 😄 

![image](https://github.com/ulises2607/Math-Magicians/assets/20537712/ad1d1632-1470-4746-8664-cbb742a502b1)